doc/configcommands.dsv

```diff
@@ -99,8 +99,13 @@ ttrss-mode|[multi/single]|multi|Configures the mode in which Tiny Tiny RSS is us
 ttrss-password|<password>|""|Configures the password for use with Tiny Tiny RSS. Double quotes should be escaped, i.e. you should write +{backslash}"+ instead of +"+.|ttrss-password "here_goesAquote:{backslash}""
 ttrss-passwordfile|<path-to-file|""|A more secure alternative to the above, by storing your password elsewhere in your system.|ttrss-passwordfile "path-to-file"
 ttrss-url|<url>|""|Configures the URL where the Tiny Tiny RSS installation you want to use resides.|ttrss-url "http://example.com/ttrss/"
+ocnews-login|<username>|""|Sets the username for use with the ownCloud instance.|ocnews-login "user"
```

Comment:
It's "to use", not "for use". I see it's written that way in the rest of the file. You can amend your patch and leave the rest as it is, and I'll edit it later; or you can amend your patch and then make another commit to fix the rest yourself.